src/te/schedule/schedule_dataflow_rewrite.cc

```diff
@@ -511,6 +511,29 @@ void InjectInline(ScheduleNode* sch, bool feature_extraction_mode) {
   std::vector<bool> changed(sch->stages.size(), false);
   std::vector<Stmt> new_hybrid_body(sch->stages.size());
   std::vector<bool> hybrid_changed(sch->stages.size(), false);
+  // (sshtin): this workaround allows to inline extern ops.
+  // All inputs for extern op should not be inlined because inlining happens
+  // before generation of TE script for particular extern op. That may lead to
```

Comment:
What do you mean by TE scripts?

Reply:
These are gen_ir() or gen_ir_1d() scripts. The invocation of these scripts may happen after inlining pass.

Reply:
How about simply dropping `script`? I don't think "TE script" is a common term. And "generation of TE" sounds ok.